test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-SddlString.Tests.ps1

```diff
@@ -0,0 +1,21 @@
+Describe "ConvertFrom-SddlString" -Tags "CI" {
+    It "Should convert without type" -Skip:($IsLinux -Or $IsOSX) {
```

Comment:
Since this cmdlet is not supported on non-Windows, you should skip them in bulk using this pattern https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk (that way if additional tests get added they get auto-skipped)